src/ReviewsServiceProvider.php

```diff
@@ -1,42 +1,25 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Tipoff\Reviews;
 
-use Illuminate\Support\Str;
 use Spatie\LaravelPackageTools\Package;
 use Spatie\LaravelPackageTools\PackageServiceProvider;
 
 class ReviewsServiceProvider extends PackageServiceProvider
 {
-    public function configurePackage(Package $package): void
+    public function boot()
     {
-        /*
-         * This class is a Package Service Provider
-         *
-         * More info: https://github.com/spatie/laravel-package-tools
-         */
-        $package
-            ->name('reviews')
-            ->hasConfigFile()
-            ->hasViews()
-            ->hasMigrations([
-                '2020_02_25_160000_create_reviews_table',
-                '2020_02_25_150000_create_keys_table',
-            ]);
+        $this->loadMigrationsFrom(__DIR__.'/../database/migrations');
+
+        parent::boot();
     }
 
-    /**
-     * Using packageBooted lifecycle hooks to override the migration file name.
-     * We want to keep the old filename for now.
-     */
-    public function packageBooted()
+    public function configurePackage(Package $package): void
     {
-        foreach ($this->package->migrationFileNames as $migrationFileName) {
-            if (! $this->migrationFileExists($migrationFileName)) {
-                $this->publishes([
-                    $this->package->basePath("/../database/migrations/{$migrationFileName}.php.stub") => database_path('migrations/' . Str::finish($migrationFileName, '.php')),
-                ], "{$this->package->name}-migrations");
-            }
-        }
+        $package
+            ->name('reviews')
+            ->hasConfigFile();
     }
 }
```

Comment:
Should we update every package to use this structure?

Would it be better to refactor to our own function of `->loadMigrations()` that handles the boot method in the chaining style of Spatie's package tools?

Reply:
Yes.  The changes in this file (along with removing `.stub` extension from the migration files themselves) changes this package from publishing migrations to loading them directly from the package (ie, method #2 in the conversation yesterday).